src/integrationTest/java/uk/gov/hmcts/reform/hmc/service/UnNotifiedHearingServiceIT.java

```diff
@@ -43,7 +43,7 @@ void testValidateUnNotifiedHearing_WithAllMandatoryFields() {
         LocalDateTime startFrom = convertDateTime(dateStr);
         UnNotifiedHearingsResponse response = unNotifiedHearingService
             .getUnNotifiedHearings("ACA2", startFrom, null,hearingStatusListed);
-        final List<String> expectedHearingIds = Arrays.asList("2100000000", "2100000001");
+        final List<String> expectedHearingIds = Arrays.asList("2100000001", "2100000000");
```

Comment:
We've seen this issue before where the results order can change and, needlessly, break tests.  Try: 
assertThatCollection(response.getHearingIds()).containsExactlyInAnyOrderElementsOf(expectedHearingIds)

Reply:
changed to assertThat